packages/sso-menu/src/SSOMenu.tsx

```diff
@@ -0,0 +1,248 @@
+    slug: 'support',
+  },
+] as const;
+
+const slugs = menuItems.map(mi => mi.slug);
+
+export const ActiveProduct = {
+  Atlas: 'atlas',
+  University: 'university',
+  Support: 'support',
+} as const;
+
+type ActiveProduct = typeof ActiveProduct[keyof typeof ActiveProduct];
+
+interface SSOMenuProps {
+  /**
+   * Object that contains information about the active user. {name: 'string', email: 'string'}
+   */
+  user: { name: string; email: string };
+
+  /**
+   * MongoDB product that is currently active: ['atlas', 'university', 'support'].
+   */
+  activeProduct: ActiveProduct;
+
+  /**
+   * Callback invoked when user logs out.
+   */
+  onLogout?: React.MouseEventHandler;
+
+  /**
+   * Callback invoked when user switches products.
+   */
+  onProductChange?: React.MouseEventHandler;
+}
+
+/**
+ * # SSOMenu
+ *
+ * ```
+<SSOMenu
+    user={{ name: 'Alex Smith', email: 'alex.smith@youwork.com' }}
+    activeProduct="atlas"
+    onLogout={() => console.log('On logout')}
+    onProductChange={() => console.log('Switching products')}
+  />
+ * ```
+ * @param props.user Object that contains information about the active user. {name: 'string', email: 'string'}
+ * @param props.activeProduct  MongoDB product that is currently active: ['atlas', 'university', 'support'].
+ * @param props.onLogout Callback invoked when user logs out.
+ * @param props.onProductChange Callback invoked when user switches products.
+ *
+ */
+function SSOMenu({
+  user: { name, email },
+  activeProduct,
+  onLogout = () => {},
+  onProductChange = () => {},
+}: SSOMenuProps) {
+  const [open, setOpen] = useState(false);
+  return (
+    <button
+      onClick={() => setOpen(curr => !curr)}
+      className={cx(menuButtonStyle, {
+        [activeMenuButtonStyle]: open,
+      })}
+    >
+      <span className={menuNameStyle}>{name}</span>
+      {open ? (
+        <Icon glyph="CaretUp" fill={uiColors.white} className={iconStyle} />
+      ) : (
+        <Icon
+          glyph="CaretDown"
+          fill={uiColors.gray.dark1}
+          className={iconStyle}
+        />
+      )}
+      <Menu open={open} setOpen={setOpen}>
+        <MenuGroup className={accountMenuGroupStyle}>
+          <h3 className={cx(nameStyle, truncate)}>{name}</h3>
+          <p className={descriptionStyle}>{email}</p>
+          <Button
+            size="small"
+            href="https://account.mongodb.com"
+            className={accountButtonStyle}
+            as="a"
+          >
+            MongoDB Account
+          </Button>
+        </MenuGroup>
+        <MenuGroup>
+          {menuItems.map(el => (
+            <MenuItem
+              onClick={onProductChange}
+              key={el.displayName}
+              active={el.slug === activeProduct}
+              className={menuItemPadding}
+              href={el.href}
+            >
+              <p className={menuItemTextStyle}>{el.displayName}</p>
+              <p className={descriptionStyle}>{el.description}</p>
```

Comment:
We shouldn't need to do this anymore with your Menu changes.

Reply:
👍